src/pages/CodeEditor/CodeEditorContainer/CompileOutput.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import Typography from "@mui/material/Typography";
+import Paper from "@mui/material/Paper";
+import { styled } from "@mui/material/styles";
+
+const CompileOutputContainer = styled("div")(
+  ({ theme }) => `
+  text-align: left;
+  height: 100%;
+  overflow-y: auto;
+`
+);
+
+const OutputPaper = styled("div")(
+  ({ theme }) => `
+  height: 80%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  box-shadow: none;
+  padding: ${theme.spacing(1)} ${theme.spacing(2)};
+  line-height: 20px;
```

Comment:
You don't have to do this in this PR but we should have standardized font sizings and component spacing.

Reply:
component spacing I think theme.spacing(x) is probably fine. The default is 8px using theme.spacing line-height should probably change though